Updated documentation for polar quadrature sets

samuelshaner opened the pull request.

wbinventor commented on docs/source/methods/track_generation.rst:
```diff
@@ -9,7 +9,7 @@ The method of characteristics uses tracks across the geometry to discretize the
 
-Cylic Tracks
+Cyclic Tracks
 ============
```
Does the underlying "=" need to be extended to the full length of this subtitle?

samuelshaner replied:
good catch - yes it does.

wbinventor commented on docs/source/methods/track_generation.rst:
```diff
@@ -80,15 +80,58 @@ The azimuthal angle quadrature set is computed based on the fraction of azimutha
+Polar Angle Quadrature
+======================
+
+In OpenMOC, there are five polar quadrature sets that couple with the standard constant-angle azimuthal quadrature set. These include equal angles, equal weights, Gauss Legendre, Leonard,and Tabuchi-Yamamoto polar quadrature sets.  The formulas for computing the angles and weights of the equal angles and equal weights quadrature sets, given below, are described in the Handbook of Nuclear Engineer ...
```
space between "Leonard," and "and"

wbinventor commented:
This looks great! Ready to merge once a few typos are addressed.

wbinventor commented:
The only other thing I can think of for this PR would be to include the
code you used to generate the polar quad plots as downloadable attachments.
This could be like the files included in the developers guide for SWIG.
What do you think @samuelshaner?
On Jan 22, 2016 7:28 PM, "sam" <notifications@github.com> wrote:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/mit-crpg/OpenMOC/pull/252
> Commit Summary
>
>    - updated documentation for polar quadrature sets
>    - Merge remote-tracking branch 'upstream/develop' into docs-polar-quad
>
> File Changes
>
>    - *A* docs/img/polar_quadrature_sets.png
>    <https://github.com/mit-crpg/OpenMOC/pull/252/files#diff-0> (0)
>    - *M* docs/source/methods/track_generation.rst
>    <https://github.com/mit-crpg/OpenMOC/pull/252/files#diff-1> (69)
>
> Patch Links:
>
>    - https://github.com/mit-crpg/OpenMOC/pull/252.patch
>    - https://github.com/mit-crpg/OpenMOC/pull/252.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/OpenMOC/pull/252>.
>

wbinventor commented:
One more thing - should we provide a few code examples showing how one can interchange the different polar quadratures or define a custom one? Or do you want to wait on that if you'll be refactoring the quadratures anyway?

samuelshaner commented:
I've written up the `plot_quadrature(...)` method that we have in ClosedMOC, so I'll just include that in this PR and then include the code for plotting the quadrature.

samuelshaner commented:
Including custom quadratures sounds like a good idea. I'll include that with this PR. It should just require minor modifications when I refactor the quadrature.

samuelshaner commented:
I have updated this PR by adding a `plot_quadrature(...)` function to the plotter and added sample code in the documentation on how to create PolarQuad objects.

wbinventor commented:
This looks good to go to me! One more thought which can be addressed here or in a future PR is that the `thisown` attribute for the `PolarQuad` should be appropriately toggled when an object is handed off to a `Solver`.

samuelshaner commented:
Isn't that already done? The `Solver::setPolarQuadrature(...)` method is in `thisown.i` and it looks like thisown is being toggled.

wbinventor commented:
Oops, I missed that when I looked earlier! Ok, this is good to go then.
On Jan 23, 2016 6:54 PM, "sam" <notifications@github.com> wrote:

> Isn't that already done? The Solver::setPolarQuadrature(...) method is in
> thisown.i and it looks like thisown is being toggled.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/OpenMOC/pull/252#issuecomment-174235367>.
>
